src/NServiceBus.RabbitMQ/Administration/QueueCreator.cs

```diff
@@ -6,13 +6,13 @@
     class QueueCreator : ICreateQueues
     {
         readonly ConnectionFactory connectionFactory;
-        readonly IRoutingTopology topology;
+        readonly IRoutingTopology routingTopology;
```

Comment:
I agree, and I do typically do renames in a separate commit, but I missed this one somehow. Didn't seem worth trying to go back and split it out once I noticed it, though.